lcls_tools/common/image/roi.py

```diff
@@ -35,33 +35,28 @@ def crop_image(self, img) -> np.ndarray:
         return img
 
 
-class RectangularROI(ROI):
-    """
-    Define a rectangular region of interest (ROI) for an image, cropping pixels outside
-    the ROI.
-    """
-    width: List[PositiveFloat]
-
-    @property
-    def bounds(self):
-        return self.width
-
-
 class EllipticalROI(ROI):
     """
     Define an elliptical region of interest (ROI) for an image.
     """
-    radius: List[PositiveFloat]
-
-    @property
-    def bounds(self):
-        return [r * 2 for r in self.radius]
+    radius: Optional[List[PositiveFloat]] = None
+    width: Optional[List[PositiveFloat]] = None
```

Comment:
Do we not get `width` and `radius` attributes for free when inheriting from the ROI base class?